manifests/agent/config.pp

```diff
@@ -2,7 +2,9 @@
 
   Ini_setting {
     path    => $puppet::conf,
+    section => 'agent',
     ensure  => 'present',
+    notify  => Service['puppet_agent'],
```

Comment:
oh, i thought it might be a problem, because in the case of cron, we don't actually declare a service for puppet-agent